File: in_Java/0215 Kth Largest Element in an Array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>((a,b) -> (b-a));
        for(int i : nums) pq.add(i);
        int res = 0;
        while(k-- != 0) res = pq.poll();
        return res;
    }
}

File: in_Python/0215 Kth Largest Element in an Array.py
class Solution(object):
    def findKthLargest(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        q = []
        for i in nums:
            heappush(q, i)
        while len(q) > k:
            heappop(q)
        return heappop(q)